### ISSUE TITLE
pySBOL remove() method causes Python to crash

### ISSUE THREAD

**Issue by kirichoi** [SynBioDex/libSBOL#171]
It seems like remove() method is crashing Python. Python unittest tests remove functions by executing something like

`doc.sequences.remove(0)`

and assert an error to be raised when the object is assessed. Has there been any changes to the function making the above expression invalid?

**Comment by bbartley** [comment]
Yes, I introduced the bug.  It's trying to delete a pointer twice.  I'll look into it.

**Comment by bbartley** [comment]
The error occurs only in Python, not libSBOL, and is introduced in libsbol.i line 174.  This is one of those annoying SWIG memory management issues, and will take some time to sort out.  For now I've commented the line out (see 4884ff50d), and the test should pass.  (It will result in a memory leak when calling the remove method, however.)

**Comment by bbartley** [comment]
Related to #140
